gramps/plugins/lib/maps/geography.py

```diff
@@ -1017,11 +1019,33 @@ def add_place_from_kml(self, menu, event, lat, lon):
                     new_place.set_title(name)
                     new_place.set_latitude(str(lat))
                     new_place.set_longitude(str(lon))
-                    try:
-                        EditPlace(self.dbstate, self.uistate, [], new_place)
-                    except WindowActiveError:
-                        pass
-        kml.destroy()
+                    return new_place
+
+                if len(points) > 5:
+                    from gramps.gui.dialog import WarningDialog
+                    WarningDialog(
+                        _('You have more than 5 places in this kml file.'),
+                        _("You will need to edit each place manually "
+                          "to add place type, ...\n"
+                          "You have %d places to edit." % len(points)),
```

Comment:
You will need to code for plural forms here.  For example, the Serbian translation will be a problem if you don't.

Reply:
What is the problem ? I don't understand.

Reply:
The word "places" may change depending on the number.  For example, in Croatian: "Imate 50 mjesta za uređivanje." and "Imate 51 mjesto za uređivanje.".  I think that this is the way that slavic languages work.